the 64 aliases were removed from public api in
https://github.com/bminor/musl/commit/25e6fee27f4a293728dd15b659170e7b9c7db9bc

since the non-64 version is just off_t 64-bit anyway, there is no need for any
of this here on musl 1.2.
--- a/libphobos/libdruntime/core/sys/posix/sys/mman.d
+++ b/libphobos/libdruntime/core/sys/posix/sys/mman.d
@@ -294,11 +294,7 @@
 }
 else version (CRuntime_Musl)
 {
-    static if (__USE_LARGEFILE64) void* mmap64(void*, size_t, int, int, int, off_t);
-    static if (__USE_FILE_OFFSET64)
-        alias mmap = mmap64;
-    else
-        void* mmap(void*, size_t, int, int, int, off_t);
+    void* mmap(void*, size_t, int, int, int, off_t);
     int munmap(void*, size_t);
 }
 else version (CRuntime_UClibc)
